### ISSUE TITLE
Question how to let elfeed--download-enclosure use curl?

### ISSUE THREAD

**Issue by QiangF** [skeeto/elfeed#311]
I am using elfeed to download rss feeds of podcasts. The proxy is set via:

    (setenv "ALL_PROXY" "socks5h://host:port")

I can view the feeds, but the mps files can't be downloaded. I guess it because
elfeed--download-enclosure doesn't use curl. If elfeed uses curl anyway, why not make
elfeed--download-enclosure use it too.

**Comment by skeeto** [comment]
Yup, elfeed--download-enclosure currently uses url-copy-file. That 
function predates elfeed-curl and was never updated to use it. There's 
no reason *not* to use elfeed-curl for this. Ideally it would be updated 
to either obey the elfeed-use-curl setting or use elfeed-with-fetch, 
which does so automatically. Somebody just needs to volunteer to make 
the change. :-)

**Comment by minad** [comment]
Implemented in https://github.com/emacs-elfeed/elfeed/commit/57f94d408258a4f07122a39a327283b3dd597a9a
